apps/site/src/app/admin/participants/components/ParticipantActionPopver.tsx

```diff
@@ -0,0 +1,26 @@
+import Popover from "@cloudscape-design/components/popover";
+import { PropsWithChildren, ReactNode } from "react";
+
+interface ParticipantActionPopoverProps {
+	content: string;
+	children?: ReactNode;
+}
```

Comment:
Nitpick/Chore: we can use the `PropsWithChildren` generic instead of manually specifying the children type